doc/help/kdb-record.md

```diff
@@ -0,0 +1,144 @@
+# kdb-record(1) -- Session recording for Elektra
+
+## SYNOPSIS
+
+`kdb record <COMMAND>`
+
+Where `<COMMAND>` is one of the following:
+
+- `start [session_name]`: start or resume a recording session.
```

Comment:
I think optionally adding a `parentKey` might be nice. So this would start a session `mysession` that only records changes that happen below `user:/foo`:

```
kdb record start --parent user:/foo mysession
```

So below the first two lines and the last line are recorded, but the third line is not recorded in the session:
```
kdb set user:/foo/bar 123
kdb set user:/foo/foo 456
kdb set user:/bar 789
kdb rm user:/foo/bar
```

Alternatively, the `parentKey` could be given to the `export` command. But restricting at `kdb record start` would avoid storing unnecessary data.

Reply:
Yes, this could easily be checked for with the already existing `keyIsBelowOrSame` function. Seems like a no-brainer. Should we also allow cascading keys?

Reply:
> Should we also allow cascading keys?

Yes, I would allow them. That makes it possible to isolate a session to the config for a single application (which might include `spec:/` keys).